docs/advanced-usage/containers/index.md

```diff
@@ -0,0 +1,5 @@
+# Container technologies
+
+The environment is encapsulated by means of “containers” such as:
```

Comment:
Usually the index pages do not contain text, just links to context pages... so this phrase could be removed.